src/modules/script.js

```diff
@@ -66,9 +66,10 @@ export const removeElement = (target) => {
   }
 };
 

```

Comment:
While you did a good job working on the project requirements I noticed you disabled eslint in your script.js file and index.js file, its is against microverse policy to disable linter because linters help us write clean code. So i would be nice if you enabled eslint in all your files. I have attached screen shots for you to see where you disabled eslint linters.

![Screen Shot 2022-12-15 at 08 05 20](https://user-images.githubusercontent.com/79506105/207785494-b1fe71b8-4a76-450f-b14d-c03ee7cd9f3a.png)


![Screen Shot 2022-12-15 at 08 04 59](https://user-images.githubusercontent.com/79506105/207785550-b486f3fb-1ae3-43f4-b928-49c31aa87cd9.png)



![Screen Shot 2022-12-15 at 08 04 41](https://user-images.githubusercontent.com/79506105/207785578-30815eec-20b7-4405-8e38-ebf047d7a3b0.png)

Lets stick to project requirements to avoid mistakes like this. Please always pay attention to detail. Thanks